osgp/platform/osgp-adapter-ws-shared/src/main/java/org/opensmartgridplatform/adapter/ws/shared/services/ResponseDataService.java

```diff
@@ -1,4 +1,4 @@
-/**
+    /**
```

Comment:
Formatting is messed up, and if this is the only change, I wonder why this file is included in this pull request at all.